Cargo.toml

```diff
@@ -47,6 +47,9 @@ cargo-util = "0.1.1"
 
 color-eyre = "0.6"
 
+prettytable = "0.10.0"
+byte-unit = "4.0.18"
```

Comment:
Let's use `bytesize` crate as it seems to be much slimmer and it is the crate that is used in `cargo`.